cumulus/pallets/xcmp-queue/src/lib.rs

```diff
@@ -478,7 +508,9 @@ impl<T: Config> Pallet<T> {
 		{
 			details
 		} else {
-			all_channels.push(OutboundChannelDetails::new(recipient));
+			all_channels
+				.try_push(OutboundChannelDetails::new(recipient))
+				.map_err(|_| MessageSendError::TooManyChannels)?;
```

Comment:
I know the error is very simple, but we could log it instead of ignoring it. What do you think?